### PR TITLE
MGMR-2420 Add bootable flag to disk in inventory

### PR DESCRIPTION
**Pull request by ori-amizur**
/cc @ronniel1 
/cc @eranco74

### CODE REVIEW

File: src/inventory/disks_test.go [hunk 1]
```diff
@@ -73,6 +74,53 @@ var _ = Describe("Disks test", func() {
 				SizeBytes: 5555,
 				Vendor:    "disk1-vendor",
 				Wwn:       "disk1-wwn",
+				Bootable:  true,
+			},
+		}))
+	})
```
Comment (eranco74) [inline src/inventory/disks_test.go:80]: What about multiple bootable disks?
That is what we actually care about.

File: src/inventory/disks.go [hunk 2]
```diff
@@ -56,6 +57,18 @@ func (d *disks) getHctl(name string) string {
 	return files[0].Name()
 }
 
+func (d *disks) getBootable(path string) bool {
+	if path == "" {
```
Comment (eranco74) [inline src/inventory/disks.go:61]: When will it get empty path?

Comment (ori-amizur) [reply src/inventory/disks.go:61]: Then we cannot figure it out

File: src/inventory/disks_test.go (hunk 1 above)
Comment (ori-amizur) [reply src/inventory/disks_test.go:80]: The information is per disk.  If we have multiple disks, then each of them has such flag

**Pull request merged**